pkgs/development/python-modules/vllm/default.nix

```diff
@@ -150,23 +183,36 @@ buildPythonPackage rec {
     ++ lib.optionals cudaSupport [
       cupy
       pynvml
+      torchWithCuda
```

Comment:
Using torchWithCuda/torchWithRocm directly in derivations doesn't work well.

At least it will be two torches in dependency tree - one is normal torch added to vllm-flash-attn, and second is this one.

It would be better for user to just have rocmSupport/cudaSupport enabled globally in nixpkgs config (`nixpkgs.config.cudaSupport = true;` in nixos / `import nixpkgs {config.cudaSupport = true;}` in flakes)

Reply:
Indeed, you are right.
My motivation was that the package was defaulting to build with rocm, but the `torch` that it would use would not be the rocm-enabled one.